.github/workflows/rebuild-shaders.yml

```diff
@@ -0,0 +1,23 @@
+name: Rebuild shaders
+
+on: workflow_dispatch
+
+jobs:
+  rebuild:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}    
+    steps:
+    - uses: msys2/setup-msys2@v2
+      with:
+        install: git make mingw-w64-x86_64-gcc mingw-w64-x86_64-python mingw-w64-x86_64-libc++    
+    - uses: actions/checkout@master
+    - name: Build
+      env:
+        MINGW64: "/mingw64"
+      run: make shaders
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: Rebuild shaders
+        repository: bgfx/shaders
```

Comment:
should not this be mamedev/mame ?

Reply:
No, this specifies the directory inside the repository to check for updated files. Anything in `bgfx/shaders` that was changed will be committed.

Reply:
Thanks, "repository" was thing that gave me wrong impression :)